frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/ClusterPolicySidebar.tsx

```diff
@@ -310,7 +311,7 @@ export function ClusterPolicySidebar(props: { data: PolicyReport }) {
         <div className={classes.body}>
             <TextContent className={classes.titleText}>
                 <Text component={TextVariants.h2}>
-                    {t('{0} identified issues').replace('{0}', policyReportViolations.length.toString())}
+                    {t(`${pluralize(policyReportViolations.length, 'identified issue')}`)}
```

Comment:
How confident are we with this Patternfly function - do we know of any current word limitations? There are a few other places we could implement this as well.

Reply:
I saw it used once already in /backend so kept same library.  But we could also make use of React's pluralize if more robust.